### PR TITLE
Added YouTubers to the index.html

### PR DESCRIPTION
**Pull request by PUJAmangale1221**

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Please fix merge conflicts and then we will have to shift the order of the youtubers by catergory.

**Comment by PUJAmangale1221** [comment]
Done, Please have a look.

File: index.html [hunk 1]
```diff
@@ -210,11 +278,42 @@ <h1>YouTubers' Birthdays</h1>
...
+				</tr>
+        <tr>
+					<td>The YouTube Philanthropist</td>
+					<td><a href="https://youtube.com/@MrBeast">MrBeast</a></td>
+					<td> 7 May, 1998
+					<td>20.9 billion</td>
+				</tr> 
+				<tr>
+					<td>Short Films</td>
+					<td><a href="https://youtube.com/@tomrworks">Timeworks</a></td>
```
Comment (KendallDoesCoding) [inline index.html:311]:
```suggestion
					<td><a href="https://youtube.com/@timeworks">Timeworks</a></td>
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Comment by PUJAmangale1221** [comment]
What should I do?

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @PUJAmangale1221.

**Pull request merged**